src/choropleth.ts

```diff
@@ -169,7 +231,7 @@ function colorizeMap(trend) {
       colorScale = colorScaleSafety;
       break;
     default:
-      console.log(`Unknown trend: ${trend} set on map`);
+      console.log(`Unknown trend: ${selectedTrend} set on map`);
```

Comment:
Remove debug console.log.

Reply:
This is there to log an error condition. I could change it to console.error, but I think it should stay.

Reply:
Yep, I didn't see that.  LGTM.